content-elements/form/checkbox/prototype/template.twig

```diff
@@ -14,15 +14,15 @@
 
     <div data-bsi-element="{{ checkboxElementId }}" class="{{ 'bsi-element-' ~ checkboxElementId }} mb-4 bsi-form-element form-check bsi-form-flex-label bsi-checkbox-spacing-visible bsi-form-switch-hide" data-bsi-element-part="{{ checkboxElementPartId }}">
         <div class="bsi-form-input-element">
```

Comment:
Du hast die Validierung in diese Klasse mit reingenommen - bitte sicherstellen, dass das keine Auswirkung hat